README.md

```diff
@@ -220,11 +220,92 @@ To run the demo, please follow these steps:
+
+Let's jump in and get this up and running in Azure. When you are finished, you will have a fully functional chatgpt app deployed to the cloud. In later steps, you'll see how to setup a pipeline and run the application.
+
+!["Screenshot of deployed chatgpt app"](assets/chatgpt.png)
+
+<sup>Screenshot of the deployed chatgpt app</sup>
+
+### Prerequisites
+
+The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
+
+- [Azure Developer CLI](https://aka.ms/azd-install)
+- [Java 17 or later](https://learn.microsoft.com/java/openjdk/install)
+- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
+- An Azure subscription with access granted to Azure OpenAI (see more [here](https://customervoice.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7en2Ais5pxKtso_Pz4b1_xUOFA5Qk1UWDRBMjg0WFhPMkIzTzhKQ1dWNyQlQCN0PWcu))
+
+### Quickstart
+
+To learn how to get started with any template, follow the steps in [this quickstart](https://learn.microsoft.com/azure/developer/azure-developer-cli/get-started?tabs=localinstall&pivots=programming-language-java) with this template(`Azure-Samples/spring-chatgpt-sample`).
+
+This quickstart will show you how to authenticate on Azure, initialize using a template, provision infrastructure and deploy code on Azure via the following commands:
+
+```bash
+# Log in to azd. Only required once per-install.
+azd auth login
+
+# Enable Azure Spring Apps feature for AZD
+azd config set alpha.springapp on
+
+# First-time project setup. Initialize a project in the current directory, using this template. 
+azd init --template Azure-Samples/spring-chatgpt-sample
+
+# Provision and deploy to Azure
+azd up
+```
+
+The template by default uses a pre-built documentation [vector store](https://asawikigpt.blob.core.windows.net/demo/doc_store.json) of the [public documents](https://github.com/MicrosoftDocs/azure-docs/tree/main/articles/spring-apps) of the Azure Spring Apps.
+
+If you want to load your own documents to the vector store, you can use the following command before running `azd up`:
+```shell
+# under the root of the project
+mvn clean package
+java -jar spring-chatgpt-sample-cli/target/spring-chatgpt-sample-cli-0.0.1-SNAPSHOT.jar --from=/<path>/<to>/<your>/<documents> --to=doc_store.json
+```
+
+### Application Architecture
+
+This application utilizes the following Azure resources:
+
+- [**Azure Spring Apps**](https://docs.microsoft.com/azure/spring-apps/) to host the application
+- [**Azure OpenAI**](https://docs.microsoft.com/azure/cognitive-services/openai/) for ChatGPT
+- [**Azure Storage**](https://docs.microsoft.com/azure/storage/) for vector store
```

Comment:
We need to clarify that the vector store is just a simple in-memory implementation. And the Azure Storage is used as a normal file volume for persistence.